frontend/src/modules/transactions/components/ExchangeRequest.tsx

```diff
@@ -111,7 +111,7 @@ export default function ExchangeRequest(): ReactElement {
         .createApiClient()
         .post(AppRoutingConstants.EXCHANGE_REQUESTS_PATH, exchangeRequest)
         .then((): void => {
-          navigate(AppRoutingConstants.HOMEPAGE);
+          navigate(`/exchange-detail/${id}/${selectedProductId}`);
```

Comment:
same question with previous? exchange detail's ID is unique so why we need product id?